installer.tpl.sh

```diff
@@ -95,6 +95,10 @@ detect_os ()
         if [ $dist != "2" ]; then
           unsupported_os
         fi
+      elif [ $os = "redos" ]; then
```

Comment:
It can be deleted if you consider my comment above.

Reply:
fixed